docs/packaging-a-RNComponent-ios.md

```diff
@@ -0,0 +1,77 @@
+---
+id: packaging-a-RNComponent-ios
+title: Packaging a react-native component
+layout: docs
+category: Guides (iOS)
+permalink: docs/packaging-a-RNComponent-ios.html
+next: native-modules-android
+---
+
+After you've written some code in an existing project and it works great
+and you want to pull it out into its own package, put it on npm, then
+install it back into your project. But wait, your component is partially
+implemented in Objective-C! How can you share this on `npm`? 
+Please read the following steps
```

Comment:
This introduction is great, but I'd suggest splitting it into smaller sentences to make it easier to read, what do you think?